### PR TITLE
Fix: Change direct link to relative link

### PR DESCRIPTION
**Pull request by lovejoy**
reading the document ,the link open a new tab and other not .and fix product_uuid to product-uuid

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubeadm.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
 * 2 GB or more of RAM per machine (any less will leave little room for your apps)
 * 2 CPUs or more 
 * Full network connectivity between all machines in the cluster (public or private network is fine)
-* Unique hostname, MAC address, and product_uuid for every node. See [here](https://kubernetes.io/docs/setup/independent/install-kubeadm/#verify-the-mac-address-and-product_uuid-are-unique-for-every-node) for more details.
+* Unique hostname, MAC address, and product_uuid for every node. See [here](/docs/tasks/tools/install-kubeadm/#verify-the-mac-address-and-product-uuid-are-unique-for-every-node) for more details.
```
[the remark this answers is not in the document]
Comment (lovejoy) [reply content/en/docs/tasks/tools/install-kubeadm.md]: fixed

**Comment by lovejoy** [comment]
ping @mistyhacks

**Comment by mistyhacks** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**